src/Umbraco.Web/Cache/DistributedCacheBinder_Handlers.cs

```diff
@@ -390,13 +390,13 @@ private void FileService_SavedTemplate(IFileService sender, SaveEventArgs<ITempl
         private void MacroService_Deleted(IMacroService sender, DeleteEventArgs<IMacro> e)
         {
             foreach (var entity in e.DeletedEntities)
-                _distributedCache.RemoveMacroCache(entity);
+                _distributedCache.RemoveMacroCache(_jsonSerializer, entity);
```

Comment:
I don't think we should have to pass in a serializer here, i think the cache refresher should have it's own serializer injected and calling code shouldn't pass one in. This is also so that some serializer requirements aren't broken since the cache refresher should serialize and deserialize always with the exact same one whereas with these APIs that may not happen.